src/test/scala/nl/knaw/dans/easy/fedoratobag/DdmSpec.scala

```diff
@@ -792,6 +792,72 @@ class DdmSpec extends TestSupportFixture with EmdSupport with AudienceSupport wi
     triedDDM.flatMap(validate) shouldBe Success(())
   }
 
+  "relation" should "not throw exceptions" in {
+    val emd = parseEmdContent(Seq(
+      emdTitle, emdCreator, emdDescription, emdDates,
+      <emd:relation>
+          <eas:isPartOf>
+              <eas:subject-title>Second Timothy: When and Where? Text and Traditions in the Subscriptions</eas:subject-title>
+          </eas:isPartOf>
+          <eas:isPartOf>
+              <eas:subject-title>Briefrapport</eas:subject-title>
+              <eas:subject-link/>
+          </eas:isPartOf>
+          <eas:isPartOf>
+              <eas:subject-title>2005-09/11</eas:subject-title>
+              <eas:subject-link/>
+          </eas:isPartOf>
+          <eas:isPartOf>
+              <eas:subject-title>blabla</eas:subject-title>
+          </eas:isPartOf>
+          <eas:references>
+              <eas:subject-title>Archeologisch onderzoek verbreding Hunzeloop Elzemaat</eas:subject-title>
+              <eas:subject-link>urn:nbn:nl:ui:13-svxg-8g</eas:subject-link>
+          </eas:references>
+      </emd:relation>,
+      emdRights,
+    ))
+    val triedDDM = DDM(emd, Seq("D13200"), abrMapping)
+    triedDDM.map(normalized) shouldBe Success(normalized(
+      <ddm:DDM xsi:schemaLocation={ schemaLocation }>
+        { ddmProfile("D13200") }
+        <ddm:dcmiMetadata>
+          <ddm:references href="http://persistent-identifier.nl/urn:nbn:nl:ui:13-svxg-8g">Archeologisch onderzoek verbreding Hunzeloop Elzemaat</ddm:references>
+          <ddm:isPartOf>Second Timothy: When and Where? Text and Traditions in the Subscriptions</ddm:isPartOf>
+          <ddm:isPartOf>Briefrapport</ddm:isPartOf>
```

Comment:
i expected this should be `dc:isPartOf`, but apparently the `ddm:isPartOf` does not require the `scheme` attribute